lib/elixir/test/elixir/kernel/warning_test.exs

```diff
@@ -269,6 +269,19 @@ defmodule Kernel.WarningTest do
     purge Sample
   end
 
+  test "unused import of one of the functions in :only" do
```

Comment:
We need to change the test above (to not use `:only`) as with this patch it's not checking the warning for bare module import.
And add `\n` to the end of the expected output.